### ISSUE TITLE
🐞 error handling

### ISSUE THREAD

**Issue by s1ntaxe770r** [gofiber/recipes#49]
**Fiber version/commit**
v 1.14.4

**Issue description**
lack of  error handling?

**Expected behavior**
panic or a 500 status if the files directory is not  present

**Steps to reproduce**
create a new fiber app with the code snippet 

**Code snippet**

```go
package main
 
import (
   "github.com/gofiber/fiber"
  "log"

 
func main() {
   app := fiber.New()

 app.Post("/",func(c *fiber.Ctx) {
	file, err := c.FormFile("file")
	if err == nil {
		c.SaveFile(file, fmt.Sprintf("./files/%s", file.Filename))
	}

})

log.Fatal(app.Listen(3000))

}
```

**Comment by CreamyMilk** [comment]
The **c.SaveFile** method returns an error of type _*os.PathError_ which you can use to panic or send a 500 status code.

```
package main
 
import (
   "github.com/gofiber/fiber"
  "log"

 
func main() {
   app := fiber.New()

 app.Post("/",func(c *fiber.Ctx) {
	file, err := c.FormFile("file")
	if err == nil {
		err = c.SaveFile(file, fmt.Sprintf("./files/%s", file.Filename))
                if err != nil {
                              // You can do the error 👷 👷 handling here
				c.Status(500).Send(err)
                               // or 
				panic(err)
			}
	}
})

log.Fatal(app.Listen(3000))

}
```
@s1ntaxe770r  Hope it work for you😊

**Comment by CreamyMilk** [comment]
fixes #49 @Fenny
